opentaskspal/src/main/java/org/dmfs/opentaskspal/predicates/Relation.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.opentaskspal.predicates;
+
+import org.dmfs.android.contentpal.predicates.DelegatingPredicate;
+import org.dmfs.tasks.contract.TaskContract;
+
+
+/**
+ * @author Gabor Keszthelyi
+ */
+public final class Relation extends DelegatingPredicate
```

Comment:
The name doesn't really express that this is a predicate. Consider calling it `IsRelation` or something like this. Same goes for `Property`.

Reply:
Update to `IsRelation` and `IsProperty`, but I also see that we don't need these currently. Shall we keep them or remove?

Reply:
Keep them.